app/test_split_integer.py

```diff
@@ -1,21 +1,67 @@
 from app.split_integer import split_integer
+import pytest
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+@pytest.mark.parametrize("value, number_of_parts, expected_result", [
+    (8, 1, [8]),
+    (6, 2, [3, 3]),
+    (17, 4, [4, 4, 4, 5]),
+    (32, 6, [5, 5, 5, 5, 6, 6])
+])
```

Comment:
Nice that you going over and beyond using parametrize here, but it is also a good practice to add id/ids for parametrized tests